### ISSUE TITLE
HR CustomDataset Finetuning

### ISSUE THREAD

**Issue by ZeVicTech** [plemeri/InSPyReNet#46]
Hi, I have a question so I'm leaving an issue.

I currently have around 9000 HR custom dataset and I am training by adding a path to the Plus_Ultra.yaml config file. (We are also training the learning dataset specified in the Plus_Ultra.yaml.)

However, The more I train the model, the lower the performance of the model.(almost 10 epoch)

May I ask you a few questions as I want to solve the cause of a problem like this?

1. Is it right to include LR datasets (ECSSD, FSS-1000, …) when creating models with 1024x1024 inputs?
2. Is there a reason to use dynamic_resize instead of static_resize for the test part of Plus_Ultra.yaml config?

If you have any other solution, I would appreciate it if you could let me know.

**Comment by plemeri** [comment]
Hi @ZeVicTech, first of all, sorry for the late reply. I cannot guarantee that I can give you the right answer since I cannot access to your custom dataset and test myself, but please make sure that your dataset annotation strategy is identical to the salient object detection task. 

For example, when your target is to generate a segmentation mask of a human hair only, then the rest of the public training datasets are not helpful for the finetuning. Try using your custom dataset only with lower learning rate, and start from our pre-trained model. You can refer to https://github.com/plemeri/InSPyReNet/issues/40 for finetuning. 

For the dynamic resize, we follow CascadePSP. It helps to produce better boundary accuracy by maintaining the aspect ratio of the image while reducing the size of the image to the plausible size for the GPU memory.

**Comment by ZeVicTech** [comment]
I'll give it a try as you advise, thank you
